backend/core/detection.py

```diff
@@ -819,6 +825,54 @@ def detect_path_type_1_hijack(
                 return (-1, "-")
             return (first_neighbor_asn, "1")
 
+        @exception_handler(log)
+        def detect_path_type_P_hijack(
+            self,
+            monitor_event: Dict,
+            prefix_node: Dict,
+            prefix_node_conf: Dict,
+            *args,
+            **kwargs
+        ) -> Tuple[int, str]:
+            """
+            Type-P hijack detection.
```

Comment:
Can you add a small description of what it returns in case there is a type P hijack?

Reply:
Done this, please check below. If the explanation is comprehensive, please resolve. Otherwise let me know to update the comment.